fmriprep/workflows/bold/base.py

```diff
@@ -381,17 +397,22 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         (bold_confounds_wf, summary, [('outputnode.confounds_list', 'confounds')]),
         (bold_reg_wf, summary, [('outputnode.fallback', 'fallback')]),
         (summary, func_reports_wf, [('out_report', 'inputnode.summary_report')]),
+        (bold_split, bold_bold_trans_wf, [
+            ('out_files', 'inputnode.bold_split')]),
     ])
 
     # bool('TooShort') == True, so check True explicitly
     if run_stc is True:
         workflow.connect([
             (bold_reference_wf, bold_stc_wf, [('outputnode.bold_file', 'inputnode.bold_file'),
```

Comment:
Just to check: STC is supposed to be run on the original BOLD, not on the HMC series?